src/smartpeak/source/ui/Widget.cpp

```diff
@@ -123,16 +123,16 @@ namespace SmartPeak
 
     // headers
     const ImGuiTableFlags table_flags = ImGuiTableFlags_Resizable | ImGuiTableFlags_Hideable |
-      //ImGuiTableFlags_Sortable | 
-      ImGuiTableFlags_Borders | ImGuiTableFlags_Scroll | ImGuiTableFlags_ScrollFreezeTopRow | ImGuiTableFlags_ScrollFreezeLeftColumn;
+      ImGuiTableFlags_Borders | ImGuiTableFlags_Scroll ;
 
     if (ImGui::BeginTable(table_id_.c_str(), headers_.size(), table_flags)) {
       // First row headers
       for (int col = 0; col < headers_.size(); col++) {
         ImGui::TableSetupColumn(headers_(col).c_str());
       }
-      ImGui::TableAutoHeaders();
-
+      ImGui::TableSetupScrollFreeze(headers_.size(), 1);
```

Comment:
I think I understand: so this line replaces the above flags then?

Reply:
Yes, however this needs some more testing as I wasn't able to load some files to see if the headers stay in-place while scrolling.

Reply:
I would recommend testing that by running the GUI and loading in a file.  Please let me know if you are having issues loading a dummy data set.  I posted several videos on the SmartPeak_user slack channel, but if we should have a quick Zoom meeting, I don't mind doing that. 

Unfortunately we do not yet have a nice integration test for the GUI that tests out these visual features, so this part still needs to be confirmed by running the GUI on a sample data set.

Reply:
I have done a couple of manual tests and the headers are frozen as previously.